scripts/remove_boilerplate.py

```diff
@@ -178,6 +187,10 @@ def parse_arguments():
                              '(default: trafilatura).')
     parser.add_argument('--boilerplate-language', '-l', default='Hungarian',
                         help='boilerplate removal language (default: Hungarian)')
+    parser.add_argument('--token-filtering', '-t', type=bool, default=False,
```

Comment:
Instead of `type` and `default`, use `action='store_true'`.

Reply:
Ok.